crates/packet_core/src/primitives/mod.rs

```diff
@@ -15,7 +15,13 @@ pub use self::str::PacketString;
 pub use self::vec::PacketVec;
 
 macro_rules! impl_var_int {
-    ($($var:ident: $base:ident => $($in:ident),+ + $($out_ty:ident = $out:ident),+);*$(;)?) => {$(
+    ($($var:ident: $base:ident => $($in:ident),+ + $($out_ty:ident),+);*$(;)?) => {$(
+        #[doc = "Transparent wrapper around an "]
+        #[doc = stringify!($base)]
+        #[doc = "."]
```

Comment:
```suggestion
        #[doc = concat!("Transparent wrapper around an ", stringify!($base), "."]
```
This might be more readable